test/aggregators/aggregators.js

```diff
@@ -47,6 +47,14 @@ describe('Aggregators', () => {
     });
   });
 
+  it('has no aggregators failing when shortName is called', () => {
+    return walkTree.then(aggregators => {
+      aggregators.forEach(aggregator => {
+        assert.doesNotThrow(_ => aggregator.shortName);
```

Comment:
I'm confirming that all aggregators define a shortName, as the report uses it for the menu and to generate the anchor, e.g. #addtohomescreen.